src/index.html

```diff
@@ -14,7 +14,7 @@ <h1 class="title">Today's To Do</h1>
         <ul class="list">
             <!-- Dynamically created list items -->
         </ul>
-        <button class="button">Clear all completed</button>
+        <button class="clear-all">Clear all completed</button>
```

Comment:
- [ ] The same for the "Clear all completed" button using the `Array.filter()`. The method is implemented but does not work. Kindly make sure this functionality is implemented correctly and works as expected.

- Please make sure the updated array of items is stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored.